middlewares/middlewares.py

```diff
@@ -67,3 +68,14 @@ def process_view(self, request, view_function, *args, **kwargs):
                     'method': request.method
                 },
             }, status=405)
+
+
+# Without this class the 'request revision' still works fine.
+# TODO: how to make it effective?
+# class BypassRevisionMiddleware(RevisionMiddleware):
+#
+#     def request_creates_revision(self, request):
+#         # Bypass the revision according to ...
+#         silent = request.META.get("HTTP_X_NOREVISION", "false")
+#         return super().request_creates_revision(request) and \
+#             silent != "true"
```

Comment:
This is to fulfill the custom requirements. For eg: this middleware allows the client to skip revision by passing a header. We don't need this for now, but if we have any custom needs, then this is needed.